cluster/examples/kubernetes/ceph/upgrade-from-v1.0-apply.yaml

```diff
@@ -7,94 +7,131 @@ metadata:
     storage-backend: ceph
     rbac.ceph.rook.io/aggregate-to-rook-ceph-global: "true"
 rules:
-- apiGroups:
```

Comment:
Did anything change in this file besides whitespace? Was it failing without the change or did your editor just add it automatically?

Reply:
the editor does it automatically. editor configured to have 2 spaces for yaml file. do I need to undo the changes?

Reply:
Revert these changes. If the editor does it automatically, disable the feature.

Reply:
@BlaineEXE @travisn leaving below data for your reference

I have seen `yamllint` issue on this file without changes, are we running `yamlllint` on yaml files?

```
yamllint upgrade-from-v1.0-create.yaml
upgrade-from-v1.0-create.yaml
  1:81      error    line too long (100 > 80 characters)  (line-length)
  15:1      error    wrong indentation: expected 2 but found 0  (indentation)
  16:3      error    wrong indentation: expected 4 but found 2  (indentation)
  18:3      error    wrong indentation: expected 4 but found 2  (indentation)
  21:3      error    wrong indentation: expected 4 but found 2  (indentation)
  38:1      error    wrong indentation: expected 2 but found 0  (indentation)
  98:1      error    wrong indentation: expected 2 but found 0  (indentation)
  99:3      error    wrong indentation: expected 4 but found 2  (indentation)
  101:3     error    wrong indentation: expected 4 but found 2  (indentation)
  103:3     error    wrong indentation: expected 4 but found 2  (indentation)
  114:3     error    wrong indentation: expected 4 but found 2  (indentation)
  116:3     error    wrong indentation: expected 4 but found 2  (indentation)
  118:3     error    wrong indentation: expected 4 but found 2  (indentation)
```

Reply:
yeah forgot about this file, reverted back create.yaml file will update it